config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -615,19 +615,25 @@ periodics:
         set -eux
         # kindinv: Kubernetes in (Rootless) Docker in (GCE) VM
         # See https://github.com/rootless-containers/kubetest2-kindinv
+        #
+        # GCE VM is used for setting up cgroup v2 and systemd.
+        # (k8s-infra-prow-build lacks cgroup v2, and the kubekins-e2e container lacks systemd)
         (cd ; GO111MODULE=on go install github.com/rootless-containers/kubetest2-kindinv@master)
+        mkdir -p -m 0700 ~/.ssh
+        cp -f "${GCE_SSH_PRIVATE_KEY_FILE}" ~/.ssh/google_compute_engine
+        cp -f "${GCE_SSH_PUBLIC_KEY_FILE}" ~/.ssh/google_compute_engine.pub
```

Comment:
It seems kubetest2 tries this internally https://github.com/kubernetes-sigs/kubetest2/blob/d7fcb799ce84ceda66c8b9b1ec8eefcbe226f293/pkg/testers/node/node.go#L194-L196 , the job failing has a more errors, are we sure this one is the fatal one and the cause it is failing?

Reply:
testers/node is not used here

Reply:
I see